pages/index.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
+  <title>Homepage</title>
 </head>
 <body>
-  
+  <section class="container">
+    <div class="header">
+      <header>
+        <h1>Arika Agnew's Personal Portfolio</h1>
+      </header>
+    </div>
+
+    <div class="main-body">
+      <p>
+        Welcome to my personal portfolio! Please choose a link to
+        the side or down below to view my portfolio and learn a 
+        little bit more about me.
+      </p>
+
+      <img src="https://eu.community.samsung.com/t5/image/serverpage/image-id/826169i43B31D8BD8121CC4/image-size/large?v=v2&px=999" alt="cute hello gif">
+    </div>
+
+    <div class="navigation-links">
+      <nav>
+        <ul>
+          <li>
+            <a href="http://127.0.0.1:5500/pages/portfolio.html">Portfolio</a>
```

Comment:
I wasn't able to load the page to your Portfolio and About. Your href attribute needs to be updated to point to where the page is located. 

We need to provide the _path_ to the resource, not the URL when you visit the page. So for example the href attribute should be "portfolio.html" because the file is in the same directory as index.html. 

If a file you're linking to is not in the same directory, then you'll need to provide the relative path to it. You may need to go up a directory and then go into a different directory like the href you provided to link your stylesheet in the head element. 

You can provide a URL to the href if you're linking to content hosted on the web (like your linkedin page or github)